### PR TITLE
Mrkt 250 pre header and other email fields

### PR DESCRIPTION
**Pull request by dkBrazz**

### CODE REVIEW

File: src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java [hunk 1]
```diff
@@ -34,15 +34,20 @@
     }
 
     @Override
-    public Email loadEmailById(int id) throws MarketoApiException {
-        List<Email> execute = httpCommandExecutor.execute(new LoadEmailById(id));
-        if (execute != null && !execute.isEmpty()) {
-            return execute.get(0);
+    public Email loadEmailById(int id, Status status) throws MarketoApiException {
```
Comment (sl-mmuradov) [inline src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java:37]: I would rename this method as far as we added a new parameter

File: src/main/java/com/smartling/marketo/sdk/rest/command/email/LoadEmailById.java [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 
+import java.util.Collections;
+import java.util.Map;
+
 public class LoadEmailById extends BaseGetCommand<Email> {
     private final int id;
+    private final Status status;
 
-    public LoadEmailById(int id) {
+    public LoadEmailById(int id, Status status) {
```
Comment (sl-mmuradov) [inline src/main/java/com/smartling/marketo/sdk/rest/command/email/LoadEmailById.java:14]: Would rename this too

File: src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java (hunk 1 above)
Comment (dimitrystd) [reply src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java:37]: As for me, it's regular method overloading.

Comment (dkBrazz) [reply src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java:37]: 1 endpoint - 1 method name
Almost all Marketo endpoints have status as a parameter, by default, it is APPROVED
I just introduced a more correct signature to be able to load draft as well, for testing and troubleshooting purposes

Comment (sl-mmuradov) [reply src/main/java/com/smartling/marketo/sdk/rest/MarketoEmailRestClient.java:37]: got it, thanks

**Pull request merged**